build/Build.Steps.cs

```diff
@@ -532,19 +530,32 @@ partial class Build
                     depsJson.RemoveDuplicatedLibraries(architectureStores);
                     depsJson.RemoveOpenTelemetryLibraries();
 
+                    // To allow roll forward for applications, like Roslyn, that target one tfm
+                    // but have a later runtime move the libraries under the original tfm folder
+                    // to the latest one.
                     if (folderRuntimeName == TargetFramework.NET6_0)
                     {
-                        // To allow roll forward for applications, like Roslyn, that target one tfm
-                        // but have a later runtime move the libraries under the original tfm folder
-                        // to the latest one.
-                        depsJson.RollFrameworkForward(TargetFramework.NET6_0, TargetFramework.NET7_0, architectureStores);
+                        depsJson.RollFrameworkForward(TargetFramework.NET6_0, TargetFramework.NET8_0, architectureStores);
+                    }
+                    else if (folderRuntimeName == TargetFramework.NET7_0 || folderRuntimeName == TargetFramework.NET8_0)
```

Comment:
Check why build  is producing 
`/store/x86/net8.0/microsoft.extensions.logging.configuration/7.0.0/lib/net7.0/Microsoft.Extensions.Logging.Configuration.dll,`

instead of 

`/store/x86/net8.0/microsoft.extensions.logging.configuration/7.0.0/lib/net8.0/Microsoft.Extensions.Logging.Configuration.dll,`

for net8.0.  @rajkumar-rangaraj, any advice here?

Reply:
Since we're using the RC version of .NET 8.0, the store folder name should be `8.0.0-rc.1.23419.4` instead of `8.0.0`.